File: send_chat_prompt.js
const openaiClient = require('./openai_client.js')

const sendChatPrompt = async ({ messages }) => {
  const completion = await openaiClient.createChatCompletion({
    model: 'gpt-3.5-turbo',
    messages
  })
  const message = completion.data.choices[0].message
  return message
}

module.exports = { sendChatPrompt }

File: send_chat_prompt.py
import openai
import os
from dotenv import load_dotenv

load_dotenv()
openai.api_key = os.environ['OPENAI_API_KEY']


def send_chat_prompt(messages):
    completion = openai.ChatCompletion.create(
        model='gpt-3.5-turbo',
        messages=messages
    )
    message = completion.choices[0].message
    return message
